modules/ivc_champva/app/controllers/ivc_champva/v1/pega_controller.rb

```diff
@@ -10,19 +10,27 @@ def update_status
         Datadog::Tracing.trace('Start PEGA Status Update') do
           data = JSON.parse(params.to_json)
 
+          tags = ['service:ivc_champva', 'function:form submission to Pega']
```

Comment:
I believe this should be
`tags = ['service:veteran-ivc-champva-forms', 'function:form submission to Pega']`

to match our service tag in `ivc_champva/v1/application_controller.rb`

Reply:
Ahh good catch

Reply:
@cloudmagic80 do you need to add the form id in the messaging like @michaelclement suggests?

Reply:
Yes I've made the changes